webserver/tests/websocket/src/main/java/io/helidon/webserver/tests/websocket/EchoService.java

```diff
@@ -54,6 +55,10 @@ public void onClose(WsSession session, int status, String reason) {
     @Override
     public Optional<Headers> onHttpUpgrade(HttpPrologue prologue, Headers headers) throws WsUpgradeException {
         WritableHeaders<?> upgradeHeaders = WritableHeaders.create();
+
+        if (!headers.get(HeaderNames.CONNECTION).get().equals("Upgrade")) {
```

Comment:
Naïve question: is case significant?

Reply:
From what I could gather, yes

Reply:
I ask because see also: https://developer.mozilla.org/en-US/docs/Web/HTTP/Protocol_upgrade_mechanism

Reply:
I see, I will switch to insensitive for the test then